health/drivers/tcp/driver.py

```diff
@@ -154,8 +155,15 @@ def main(es, latest_aggregated_ts=None):
 
     for interval in intervals:
         body = get_request(interval)
-        resp = requests.post("%s/_search?search_type=count" % es,
-                             data=json.dumps(body)).json()
+        resp = requests.post("%s/_search" % es,
```

Comment:
there is no more search_type=count in es5, instead it has a count API, that does not support aggs. https://www.elastic.co/guide/en/elasticsearch/reference/current/search-count.html

so I guess we need to use https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-metrics-valuecount-aggregation.html instead?